src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import Box from '@mui/material/Box';
+import RegisterLoginHeader from '../RegisterLoginHeader/RegisterLoginHeader';
```

Comment:
Please consider changing the imports to absolute ones. `src` is treated as a starting point and in the case of the import above the absolute import is `@/components/RegisterLoginHeader/RegisterLoginHeader`.